src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,98 @@
+package main.java.duke;
+
+import main.java.duke.command.*;
```

Comment:
Imported classes should be listed explicitly.

Reply:
If you are using IDEs such as IntelliJ, perhaps you can consider setting your preferences to not auto-optimise the import statements to avoid the IDE from replacing you import statements which seems to show you were previously aware not to use the wildcard import statement! :)